Nomster/CreateSuggestionViewController.swift

```diff
@@ -86,11 +86,11 @@ class CreateSuggestionViewController : UITableViewController {
         }
         
         // Dummy values
-        let image = "nomster-logo-large"
+        let image = ""
```

Comment:
The problem is, with this change, we need a remote server all-the-time. No more 'mocking'. We should find a reliable server to host it.